locust_plugins/listeners.py

```diff
@@ -79,6 +77,13 @@ def __init__(
         events.spawning_complete.add_listener(self.spawning_complete)
         atexit.register(self.log_stop_test_run)
 
+        if is_worker(self.env):
```

Comment:
Probably this if statement is not needed.

Reply:
done